src/backends/kafka/mod.rs

```diff
@@ -23,6 +23,16 @@ pub struct KafkaPayload {
     pub payload: Option<Vec<u8>>,
 }
 
+#[derive(PartialEq)]
+enum KafkaConsumerState {
+    NotSubscribed,
```

Comment:
NotSubscribed is new and doesn't exist in the Python library. It's the initial state before subscribe() is called.